src/Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -55,11 +56,40 @@ public NodeSettings(Network innerNetwork = null, ProtocolVersion protocolVersion
             this.LoggerFactory.AddConsoleWithFilters();
             this.LoggerFactory.AddNLog();
             this.Logger = this.LoggerFactory.CreateLogger(typeof(NodeSettings).FullName);
+
+            // TODO: Temporary until Build method updated to call LoadConfiguration
+            loadConfiguration = true;
+
+            // Load arguments or configuration from .ctor?
+            if (args != null || loadConfiguration)
```

Comment:
We can probably remove this now, I think we always want to call the code included in the `if`.

Reply:
There should be no effect in removing it. Done.